rtloader/common/builtins/datadog_agent.c

```diff
@@ -455,7 +467,7 @@ static PyObject *set_external_tags(PyObject *self, PyObject *args)
         free(tags);
     }
 
-done:
+error:
```

Comment:
I'd rather call this `done` since it's not only intended to be executed on error